docs/sections/user_guide/cli/drivers/shared/schema-options.rst

```diff
@@ -0,0 +1,9 @@
+* Specifying the ``--show-schema`` flag, with no other options, prints the driver's schema:
+
+  .. literalinclude:: show-schema.cmd
+     :language: text
+     :emphasize-lines: 1
+  .. literalinclude:: show-schema.out
+     :language: text
+
+* Use the ``--schema-file`` option to specify a custom :json-schema:`JSON Schema<>` file with which to validate the driver config. A custom schema could range in complexity from the simplest, most permissive schema, ``{}``, to one based on the built-in schema shown by ``--show-schema``.
```

Comment:
In addition to factoring out the `--show-schema` info from each driver, this shared fragment now adds info on the `--schema-file` option.